README.md

```diff
@@ -16,21 +22,25 @@
 imgex is [available in Hex](https://hex.pm/packages/imgex), the package can be
 installed as:
 
-  1. Add imgex to your list of dependencies in `mix.exs`:
+Add `:imgex` to your list of dependencies in `mix.exs`:
 
-     ```elixir
-     def deps do
-       [{:imgex, "~> 0.2.0"}]
-     end
-     ```
+```elixir
+def deps do
+  [
+    {:imgex, "~> 0.2.0"}
+  ]
+end
+```
 
-  2. Ensure imgex is started before your application:
+Ensure `:imgex` is started before your application:
```

Comment:
This section should be removed because it is only necessary for Elixir <= 1.4 which is quite old at this point

Reply:
@axelson 
Should we bump the minimum Elixir version requirement as well? And to which version?

```
 22       app: :imgex,
 23       version: @version,
 24       elixir: "~> 1.1",
```

Reply:
I'm not the maintainer but I think it would be good to bump it to at least 1.5.

Reply:
Noted. We'll let @ianwalter decides on this then.

Reply:
Okay, I'm now a maintainer and I think we should set the minimum elixir version to 1.5 and remove this section because it will no longer be necessary.